.evergreen/config.yml

```diff
@@ -138,8 +138,7 @@ functions:
           # repo will update the driver's go.mod/go.sum files. This puts the contents of go.mod/go.sum out of sync with
           # the vendor directory and tests fail to compile due to inconsistent vendoring.
           cd $(mktemp -d)
-          go get -u golang.org/x/lint/golint
-          go get -u github.com/kisielk/errcheck
+          go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.42.0
```

Comment:
Are you grabbing `v1.42.0` for a particular reason?

Reply:
Decided to go with the latest version. Also updated to use `go install` instead of `go get`.